quickstarts/dbmarlin/config.yml

```diff
@@ -0,0 +1,61 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: dbmarlin
+
+# Displayed in the UI (required)
+title: DBmarlin
+
+# Long-form description of the quickstart (required)
+description: |
+  The DBmarlin quickstart allows you to get visibility into the performance and availability of your database services from within New Relic.
+
+  The installation instructions for the quickstart are available at https://newrelic.com/instant-availability/dbmarlin and https://docs.dbmarlin.com/docs/integrations/newrelic.
+
+  You will need to install DBmarlin first. Full installation instructions for DBmarlin are available at https://docs.dbmarlin.com/docs/DBmarlin/introduction.
+
+  # About DBmarlin
+  DBmarlin monitors database performance, tracks changes and helps your databases run fast.
+
+  It provides an easy-to-use and consistent way to monitor CockroachDB, DB2, MariaDB, MySQL, Oracle, PostgreSQL and SQL Server, self-hosted or in the cloud.
+
+  It captures SQL statement text and wait events so you can see exactly where the time is spent executing SQL within your database.
+
+  It auto-detects changes to schema objects, database parameters and execution plans so you can see their impact on performance.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  View database performance from within New Relic with this integration with DBmarlin.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - DBmarlin
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - database
+  - performance
+  - availability
+  - cockroach
+  - db2
+  - mysql
+  - maria
+  - oracle
+  - postgres
+  - sql
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - infra-agent-standard
```

Comment:
This install plan doesn't exist. If you're looking to install the infrastructure agent, can I suggest the [targeted infrastructure install plan](https://github.com/newrelic/newrelic-quickstarts/blob/main/install/infrastructure/infra-agent-targeted/install.yml)?